src/main/java/lt/vu/trip/controller/AuthController.java

```diff
@@ -50,19 +35,4 @@ public ResponseEntity login(@RequestBody AuthenticationRequest data) {
 			throw new BadCredentialsException("Invalid username/password supplied");
 		}
 	}
-
-	@PostMapping("/register")
```

Comment:
kodėl perkėlei šitą į demo?

Reply:
mums nereikia registracijos - vartotojus pridės adminai. Čia tik šiaip demo tegu būna šiukšlinėlis, nes patogu testuojant taip susikurti nauja useri